Tools/autotest/arducopter.py

```diff
@@ -3639,8 +3639,8 @@ def test_guided_local_velocity_target(self, vx, vy, vz_up, timeout=3):
             self.progress("Received local target: %s" % str(m))
 
         # Check the last received message
-        if not (m.type_mask == 0xFFC7 or m.type_mask == 0x0FC7):
-            raise NotAchievedException("Did not receive proper mask: expected=65479 or 4039, got=%u" % m.type_mask)
+        if not (m.type_mask == 0xFE07 or m.type_mask == 0x0E07):
```

Comment:
I'm surprised this change to the autotests is required..

Reply:
We are now reporting Acceleration too so this changed the masks.